modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -188,30 +188,36 @@ Please consider alternatives to using this service, such as independently verify
 
 .Procedure: Installing an Errata Service
 
-. On the {productname} Server, from the command prompt, as root, add the ``sle-module-development-tools`` module:
+. On the {productname} Server host, from the command prompt, as root, add the [systemitem]``sle-module-development-tools`` module:
 +
 ----
-SUSEConnect --product sle-module-development-tools/15.2/x86_64
+mgrctl exec -- SUSEConnect --product sle-module-development-tools/15.5/x86_64
 ----
 . Install errata service dependencies:
 +
 ----
-zypper in  perl-Text-Unidecode
+transactional-update pkg install perl-Text-Unidecode
```

Comment:
Remember that after any action with transaction-update, we must reboot the system to make the change real.

Reply:
This would be fine to run the script (and the cron job) on the host.